frontend/src/pages/myPostsPage.jsx

```diff
@@ -30,9 +22,30 @@ const IconText = ({ icon, text }) => (
 
 class myPostsPage extends React.Component {
   constructor(props){
-    super(props)
-    console.log(this.props)
+    super(props);
+    this.addPostsIntoList = this.addPostsIntoList.bind(this);
+  }
+
+  componentDidMount = () =>{
+    this.props.getCurAuthorPosts();
+    listData = []
+    this.addPostsIntoList(this.props.post);
+  }
+
+  addPostsIntoList = (posts) => {
+    for (let i = 0; i < posts.length; i++) {
+      listData.push({
+        href: `${posts[i]['id']}`,
+        title: `${posts[i]['title']}`,
+        avatar: 'https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png',
```

Comment:
is avatar needed? I dont think profile handles user images

Reply:
I am leaving there for UI beautify mostly, simply plain texts looks ugly